packages/api/tests/endpoints/test-rules.js

```diff
@@ -449,22 +666,96 @@ test('PUT returns 400 for name mismatch between params and payload',
     t.falsy(record);
   });
 
+test('PUT replaces onetime rule for rerun', async (t) => {
+  const expectedRule = {
+    ...omit(testRule, ['queueUrl', 'provider', 'collection']),
+    state: 'ENABLED',
+  };
+  expectedRule.action = 'rerun';
+
+  await request(app)
+    .put(`/rules/${testRule.name}`)
+    .set('Accept', 'application/json')
+    .set('Authorization', `Bearer ${jwtAuthToken}`)
+    .send(expectedRule)
+    .expect(200);
+
+  const { body: actualRule } = await request(app)
+    .get(`/rules/${testRule.name}`)
+    .set('Accept', 'application/json')
+    .set('Authorization', `Bearer ${jwtAuthToken}`)
+    .expect(200);
+
+  t.truthy(actualRule);
+});
+
+test.serial('PUT does not insert into RDS if replacing onetime rule for rerun fails', async (t) => {
```

Comment:
A PUT request with an action of `rerun` doesn't do any writing to Dynamo or Postgres that I can see, so what is this testing?

Reply:
Hm I can see how that is confusing because my test statement and setup doesn't make sense for what I intended. What I intended to do was write a test to ensure that no record to written to postgres when a replacing rule's action is 'rerun`.

Reply:
I guess I can just combine tests

Reply:
I realize now that this test case is actually covered. I'll remove it

Reply:
[`717d0e5` (#2012)](https://github.com/nasa/cumulus/pull/2012/commits/717d0e50dc816dd503b437d92a583a563a6c0766)

Reply:
What else is covering it?